en/installconfig-lxd-install.md

```diff
@@ -4,16 +4,25 @@ TODO:  Text needs a review
 
 # Install with LXD
 
+[LXD][link-lxd] is a next generation system container manager for Linux
```

Comment:
I know it's what upstream says but it sounds both extravagant and vague to me.

Reply:
I guess I put it there more for consistency. I did make an attempt to tone it down though.